scripts/runAccTests.bat

```diff
@@ -0,0 +1,104 @@
+@echo off
+setlocal
+
+:: As of `go-dockerclient` v1.2.0, the default endpoint to the Docker daemon
+:: is a UNIX socket.  We need to force it to use the Windows named pipe when
+:: running against Docker for Windows.
+set DOCKER_HOST=npipe:////.//pipe//docker_engine
+
+:: Note: quoting these values breaks the tests!
+set DOCKER_REGISTRY_ADDRESS=127.0.0.1:15000
+set DOCKER_REGISTRY_USER=testuser
+set DOCKER_REGISTRY_PASS=testpwd
+set DOCKER_PRIVATE_IMAGE=127.0.0.1:15000/tftest-service:v1
+set TF_ACC=1
+
+call:setup
+if %ErrorLevel% neq 0 (
+  call:print "Failed to set up acceptance test fixtures."
+  exit /b %ErrorLevel%
+)
+
+call:run
+if %ErrorLevel% neq 0 (
+  call:print "Acceptance tests failed."
+  set outcome=1
+) else (
+  set outcome=0
+)
+
+call:cleanup
+if %ErrorLevel% neq 0 (
+  call:print "Failed to clean up acceptance test fixtures."
+  exit /b %ErrorLevel%
+)
+
+exit /b %outcome%
+
+
+:print
+  if "%~1" == "" (
+    echo.
+  ) else (
+    echo %~1
+  ) 
+  exit /b 0
+
+
+:log
+  call:print ""
+  call:print "##################################"
+  call:print "-------- %~1"
+  call:print "##################################"
+  exit /b 0
+
+
+:setup
+  call:log "setup"
+  call %~dp0testing\setup_private_registry.bat
+  exit /b %ErrorLevel%
+
+
+:run
+  call:log "run"
+  call go test ./docker -v -timeout 120m
+  exit /b %ErrorLevel%
+
+
+:cleanup
+  call:log "cleanup"
+  call:print "### unsetted env ###"
+  for /F %%p in ('docker container ls -f "name=private_registry" -q') do (
+    call docker stop %%p
+    call docker rm -f -v %%p
+  )
+  call:print "### stopped private registry ###"
+  rmdir /q /s %~dp0testing\auth
+  rmdir /q /s %~dp0testing\certs
+  call:print "### removed auth and certs ###"
+  for %%r in ("container" "volume") do (
+    call docker %%r ls -f "name=tftest-" -q
+    for /F %%i in ('docker %%r ls -f "name=tf-test" -q') do (
```

Comment:
@mavogel While running this script repeatedly on my machine, I noticed two things:

1. A container named `tf-test` was left around after each run.  This container's named does not match the `tftest-` pattern that is used to find dangling containers to delete.
2. The `docker rm` commands don't use `-v`.  This causes some volumes to stick around and it sometimes causes an issue where Docker will refuse to create new mount points because "`/host_mnt/c` is already mounted".

I fixed them in these batch scripts so I can now repeat the test runs, but since they are basically a transliteration of the `.sh` scripts, it's likely the cleanup bugs exist there too.